sequence/arguments/api.graphql

```diff
@@ -57,23 +60,21 @@ type Query {
             }
             """
         )
-    # A sequence call that sequences converCityOrLocaleToLatLong and weather
-    # this allows for a cityOrLocale argument to be used to fetch weather data by lat/lon
-    # we have to use an arguments setting for the second query, because   
-    # query arguments only flow to the first query in the sequence
-    # but we need units to flow to the second query
-    # we are saying that the argument named units in the query weather 
-    # gets its value from the argument units in the enclosing query
-    # In addition to the arguments explictly passed down to the second query,
-    # it also gets the output of the first query
-    # in this case, the first query returns a type Coord, and therefore its fields lat and lon are
-    # available to the second query.
+    # Below is a sequence call that sequences `convertCityOrLocaleToLatLong` and `weather`.
```

Comment:
Do we use the term "enclosing query" elsewhere, because that's not really what is going on. 

When documenting `@sequence` I try to use these terms:

- "sequence field" - the field annotated with `@sequence`, not "enclosing query"
- "sequence steps" - a step in the sequence, which invokes a field from `Query` or `Mutation`. Not "first query", since query is an overloaded term in graphql and it tends to lead to confusion.

Here `weatherByCityOrLocale` is the sequence field, it has field arguments `cityOrLocale` and `units`. These field arguments are available a step in the sequence by using `{name:"step-arg-name" argument:"sequence-field-arg-name}` in arguments.